layouts/partials/tabs.html

```diff
@@ -295,22 +295,72 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
         </li>
 
         <li id="tab-store" class="cd-tabs__panel text-component">
-        <p>Machine Learning ... .</p>
-
-        <p>
-	This tab is work in progress.  Just parking some content here for now.
-        <a href="https://www.tensorflow.org">Tensorflow</a>
-        framework helps to define and run computations involving tensors that have
-        broad applications in Speech and image recognition, Text-based
-        applications, Time-series analysis and Video detection.
-	
-	Other suggestions for Machine Learning:
-
-        Visualization: Yellowbrick, Eli5
-        ML libraries: XGBoost, LightGBM, CatBoost
-        AI libraries: PyTorch, MXNet
-        
-	</p>
+       	   <div class="grid-container">
+	      <div>
+		      <a href="https://ai.googleblog.com/2016/12/open-sourcing-embedding-projector-tool.html"> <img src="images/content_images/ml_img/pytorch-ml-anim.gif" alt="pytorch figure" align="middle" style="border-radius: 10px"></a> 
+              </div>
+	      <div>
+                 <p>
+		 <b>Machine learning</b> (ML) enables computers to learn using
+		    data, without having to be explicitly programmed.
+		    <b>NumPy</b> is the foundation of all data pre-processing
```

Comment:
This is not quite true. Perhaps better to say that these libraries all have data structures that are inspired by NumPy.